includes/class-wp-job-manager-post-types.php

```diff
@@ -659,6 +675,62 @@ public function job_feed() {
 		remove_filter( 'posts_search', 'get_job_listings_keyword_search' );
 	}
 
+	/**
+	 * Get filled jobs.
+	 *
+	 * @return array
+	 */
+	public function get_filled_job_listings(): array {
+		if ( ! get_option( 'job_manager_hide_filled_positions' ) ) {
+			return [];
+		}
```

Comment:
I think this logic is done in the _usage_ of this method. It doesn't quite match the described purpose of the method.

Reply:
Good point, and after further testing, the method returns an empty array anyway if there are no posts ID, so this check is redundant.

Committed https://github.com/Automattic/WP-Job-Manager/pull/2449/commits/3a210fb7152c0fb9a40c5393e60c33b13ee06db1